topologies/datacenter-latest/labguides/source/mlag-with-cvp.rst

```diff
@@ -0,0 +1,116 @@
+MLAG
+====
+
+.. image:: images/cvp-mlag/mlag_1.png
+   :align: center
+
+.. note:: Did you know the “mlag” script is composed of Python code that
```

Comment:
@jrecchia1029 Can probably remove this note as the user isn't leveraging the login.py code to configure the lab.